cogs/calendar.py

```diff
@@ -1,5 +1,6 @@
 import os
 import datetime
+import pytz
```

Comment:
REQ: we can remove this since it's unused

Reply:
True lol